src/cmd/services/m3aggregator/config/aggregator.go

```diff
@@ -803,6 +803,7 @@ type flushManagerConfiguration struct {
 	NumWorkersPerCPU float64 `yaml:"numWorkersPerCPU" validate:"min=0.0,max=1.0"`
 
 	// How frequently the flush times are persisted.
+	// If unset or set to 0, this will update kv flush times after every flush.
```

Comment:
this only downside to this is it's not backwards compatible. previously 0 would be the default of 10s. This is probably ok, since 0 is a saner default value IMO

Reply:
Yeah that was my thinking, I didn't really want to add another boolean on/off feature to this in an already bloated config

Reply:
Dropping this again :+1: